mrp_unbuild_rebuild_variant/views/mrp_unbuild_rebuild_variant.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2020 Camptocamp SA
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+    <record id="mrp_unbuild_rebuild_variant_view_form" model="ir.ui.view">
+      <field name="name">mrp.unbuild.rebuild.variant.form</field>
+      <field name="model">mrp.unbuild.rebuild.variant</field>
+      <field name="arch" type="xml">
+        <form string="Unbuild Rebuild Variants">
+            <header>
+                <field name="state"/>
+                <button name="process" string="Unbuild" type="object" states="draft" class="oe_highlight"/>
+            </header>
+            <sheet>
+                <group>
+                    <field name="name"/>
+                    <field name="quantity"/>
+                </group>
+                <group>                  
+                    <group name="Unbuild">
+                        <field name="unbuild_product_id" context="{'unbuild_rebuild': True}"/>
+                        <field name="unbuild_lot_id"/>
```

Comment:
This should be made invisible if the product is not tracked (through a related field)